docs/rfcs/relationship-length-validation.md

```diff
@@ -0,0 +1,43 @@
+# Relationship Length Validation
+
+## Problem
+
+With the [RFC on relationship cardinality](https://github.com/neo4j/graphql/blob/dev/docs/rfcs/rfc-003-relationship-cardinality.md), we now have the ability to validate one-to-one relationships with Cypher. We cant however, due to the fact that GraphQL doesn't have any inbuilt mechanism to represent a range for related nodes, allow users to specify for example: "Movies must have at least one Actor".
+
+## Proposed Solution
+
+Expose new `validation` parameter to the `@relationship` directive, where the following can be specified:
+
+1. `min`
+2. `max`
+3. `*_GT`
+4. `*_GTE`
+5. `*_LT`
+6. `*_LTE`
+7. `EQUAL`
```

Comment:
We don't need all of these arguments, as `max` is actually equivalent to `LT` - you could actually just have `min` and `max` - a few examples:

- Must have count of 3 - `min: 3, max: 3`
- Maximum of 3 - `max: 3`
- Greater than 1, less than or equal to 5 - `min: 1, max: 5`